src/pages/detail.html

```diff
@@ -0,0 +1,551 @@
+                <img src="../../assets/images/buy-most.png" alt="" class="mr-4">
+                <div class="">
+                  <p class="mb-3 font-semibold">Mỹ phẩm châu Âu</p>
+                  <div class="flex items-center text-sm justify-between mb-3">
+                    <div class="mr-3">
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                    </div>
+                    <span>( 4 <span data-i18n="detail.buy_turn"></span> )</span>
+                  </div>
+                  <div class="italic font-bold ">355.000đ</div>
+                </div>
+              </div>
+              <div class="flex mt-8 pb-6 border-b border-dashed">
+                <img src="../../assets/images/buy-most.png" alt="" class="mr-4">
+                <div class="">
+                  <p class="mb-3 font-semibold">Mỹ phẩm châu Âu</p>
+                  <div class="flex items-center text-sm justify-between mb-3">
+                    <div class="mr-3">
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                    </div>
+                    <span>( 4 <span data-i18n="detail.buy_turn"></span> )</span>
+                  </div>
+                  <div class="italic font-bold ">355.000đ</div>
+                </div>
+              </div>
+              <div class="flex mt-8 pb-6 ">
+                <img src="../../assets/images/buy-most.png" alt="" class="mr-4">
+                <div class="">
+                  <p class="mb-3 font-semibold">Mỹ phẩm châu Âu</p>
+                  <div class="flex items-center text-sm justify-between mb-3">
+                    <div class="mr-3">
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                      <i class="fa-solid fa-heart"></i>
+                    </div>
+                    <span>( 4 <span data-i18n="detail.buy_turn"></span> )</span>
+                  </div>
+                  <div class="italic font-bold ">355.000đ</div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div id="footer"></div>
+  </div>
+  <script src="../../assets/js/loadlayouts.js" type="module"></script>
+  <script src="../../assets/js/detail.js"></script>
+  <script src="../../assets/js/i18n.js"></script>
+  <script src="../../assets/js/breadcrumb.js"></script>
```

Comment:
```suggestion
  <script src="../../assets/js/detail.js"></script>
  <script src="../../assets/js/breadcrumb.js"></script>
  <script src="../../assets/js/i18n.js"></script>
```
Để i18n đứng cuối nha